Examples/UIExplorer/ActivityIndicatorExample.js

```diff
@@ -15,34 +15,43 @@
  */
 'use strict';
 
-const React = require('react');
-const ReactNative = require('react-native');
-const {
-  ActivityIndicator,
-  StyleSheet,
-  View,
-} = ReactNative;
-const TimerMixin = require('react-timer-mixin');
+import React, { Component } from 'react';
+import { ActivityIndicator, StyleSheet, View } from 'react-native';
 
-const ToggleAnimatingActivityIndicator = React.createClass({
-  mixins: [TimerMixin],
+/**
+ * Optional Flowtype state and timer types definition
+ */
+type StateT = { animating: boolean; };
+type TimerT = number;
+
+class ToggleAnimatingActivityIndicator extends Component {
+  /**
+   * Optional Flowtype state and timer types
+   */
+  state: StateT;
```

Comment:
Nit: Can you just remove the `T` suffix from flow type names as we don't use this convention in the codebase.

Then this looks good to me!